src/Filesystem/Dir.php

```diff
@@ -139,23 +139,31 @@ public static function files(
 
 	/**
 	 * Read the directory and all subdirectories
+	 *
+	 * @todo Remove support for `$ignore = null` in a major release
+	 * @param array|null $ignore Passing null is deprecated.
```

Comment:
```suggestion
	 * @param array|false|null $ignore Passing null is deprecated.
```

Reply:
We should probably change this in `Dir::copy()` right away as well